lib/install/vchlog/vchlogger.go

```diff
@@ -0,0 +1,42 @@
+package vchlog
+
+import (
+	"github.com/vmware/govmomi/object"
+	"context"
+	"path"
+)
+
+
+type VCHCreatedSignal struct {
+	Datastore *object.Datastore
+	LogFileName string
+	Contex context.Context
```

Comment:
s/Contex/Context

Reply:
oops thanks

Reply:
sry for the uncommented files: the design is not finalized yet